src/main/java/org/opentripplanner/transit/service/StopModel.java

```diff
@@ -36,293 +29,246 @@ public class StopModel implements Serializable {
 
   private static final Logger LOG = LoggerFactory.getLogger(StopModel.class);
 
-  /** Parent stops **/
-  private final Map<FeedScopedId, Station> stationById = new HashMap<>();
-  /**
-   * Optional level above parent stops (only supported in NeTEx)
-   */
-  private final Map<FeedScopedId, MultiModalStation> multiModalStationById = new HashMap<>();
-  /**
-   * Optional grouping that can contain both stations and multimodal stations (only supported in
-   * NeTEx)
-   */
-  private final Map<FeedScopedId, GroupOfStations> groupOfStationsById = new HashMap<>();
-
-  private final Map<FeedScopedId, TransitStopVertex> transitStopVertices = new HashMap<>();
+  private final Map<FeedScopedId, Stop> regularStopsById;
+  private final Map<FeedScopedId, Station> stationById;
```

Comment:
Should these be plural or not (`regularStopsById` vs `stationById`)?

Reply:
I thing the `stationById` is correct. We map one station pr Id.

Reply:
`stationById` is consistent with earlier practice, so I will go for that - I have done this for all places related to stops.